samples/README.md

```diff
@@ -20,30 +21,31 @@ It is recommended that you understand the developer workflow for Mystikos by rea
 ## SGX execution targets
 
 The samples below demonstate how to run Mystikos samples using the SGX execution target. To learn more about the SGX execution target, please refer to 
-the [documentation for the targets](./doc/user-getting-started.md#understand-mystikos-execution-targets).
+the [documentation for the targets](../doc/user-getting-started.md#understand-mystikos-execution-targets).
 
 ## Samples
 
 The following samples demonstate how Mystikos can be used to run applications in SGX.
 
-[Helloworld](./helloworld)
+* [Helloworld](./helloworld)
 Demonstates how to build and run the simples C application in Mystikos.
 Takes the developer through the steps of build a cpio application, an ext2 application, signing and packaging.
 
-[docker_aks](./docker_aks)
+* [docker_aks](./docker_aks)
 Demonstrates how to run a  Mystikos application from within a docker container either locally or in Azure Kubernetes Service.
 
-[gencreds](./gencreds)
-Shows an advanced developer how to write a TEE aware sample that can get a self signed certificate which can be used for attested_tls.
+* TEE Aware applications
+    - [gencreds](./TEE_Aware/gencreds) Shows an advanced developer how to write a TEE aware sample in C that can get a self signed certificate which can be used for attested_tls.
+    - [dotNET](./TEE_Aware/dotNET) Shows a developer how to get started with dotNET applications for Mystikos. It also shows how a dotNET application can retrieve TEE credentials for use.
```

Comment:
The name is more commonly spelled as "dotnet" or ".NET".

Reply:
Thanks. Changed to dotnet